### PR TITLE
Fix #1135 - lumberaxe not breaking wood 

### LINKED ISSUE #1135: lumberaxe doesnt break stripped wood and wood
## Description (Required)
when you break stripped wood or normal wood with a lumberaxe, it doesnt break the tree it registers it as not a wood type like it breaks as normal wood.

## Steps to reproduce the Issue (Required)
place down or find a custom tree that uses it
break the wood / stripped wood with the lumberaxe

## Expected behavior (Required)
expected behavior is to see it break all the connected wood, stripped wood, logs and stripped logs

## Environment (Required)

 - Minecraft Version: 1.14.4
 - Slimefun Version: 269
 - CS-CoreLib Version: 76

### PR DESCRIPTION
**Pull request by WalshyDev**
## Description
Fixes #1135 by looking for wood.

## Changes
Use the new CS-CoreLib Vein and remove all logs + wood. Also make sure to check perm on the blocks.

## Related Issues
Resolves #1135

## Testability

- [X] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -133,7 +133,7 @@
         <dependency>
   			<groupId>com.github.thebusybiscuit</groupId>
   			<artifactId>CS-CoreLib2</artifactId>
-  			<version>0.5.6</version>
+  			<version>f492af0f15</version>
```
Comment (TheBusyBiscuit) [inline pom.xml]: Sorry, only gonna do proper releases on that.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java [hunk 2]
```diff
@@ -24,32 +23,42 @@
...
-
-					if (logs.contains(e.getBlock().getLocation())) logs.remove(e.getBlock().getLocation());
-					for (Location b: logs) {
-						if (SlimefunPlugin.getProtectionManager().hasPermission(e.getPlayer(), b, ProtectableAction.BREAK_BLOCK)) {
-							b.getWorld().playEffect(b, Effect.STEP_SOUND, b.getBlock().getType());
-							
-							for (ItemStack drop: b.getBlock().getDrops()) {
-								b.getWorld().dropItemNaturally(b, drop);
+				if (MaterialCollections.contains(e.getBlock().getType(), MaterialCollections.getAllLogs())
+					|| MaterialCollections.contains(e.getBlock().getType(), MaterialCollections.getAllWood())
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java]: Please hold onto doing that, I am actually going to push an Update to MaterialCollections that will make this a lot easier.

File: pom.xml (hunk 1 above)
Comment (WalshyDev) [reply pom.xml]: REEEE

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java (hunk 2 above)
Comment (WalshyDev) [reply src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java]: Thank god

**Pull request closed without merging**